ui/apps/platform/src/Components/PatternFly/EmptyStateTemplate/EmptyStateTemplate.tsx

```diff
@@ -12,16 +12,18 @@ type EmptyStateTemplateProps = {
     children?: ReactNode;
     title: string;
     headingLevel: 'h1' | 'h2' | 'h3' | 'h4';
+    icon?: React.ComponentType;
```

Comment:
In this case I had to use `React.ComponentType` since the `icon` prop is being passed to PF's `EmptyStateIcon`, which expects an uninitialized Function/Class component that it renders itself. The `TileContent` component that we have is expecting a `ReactElement` that has already been rendered.